src/test/kotlin/no/nav/tilleggsstonader/sak/migrering/routing/SkjemaRoutingIntegrationTest.kt

```diff
@@ -103,6 +105,30 @@ class SkjemaRoutingIntegrationTest(
             assertThat(routingHarBlittLagret()).isFalse()
         }
 
+        @Test
+        fun `skal route til gammel løsning hvis person har AAP men fortrolig adresse`() {
+            mockMaksAntallSomKanRoutesPåDagligReise(maksAntall = 10)
+            mockAapVedtak(erAktivt = true)
+            mockPersonMedAdressebeskyttelse(AdressebeskyttelseGradering.FORTROLIG)
+
+            val routingSjekk = kall.søknadRouting.skjemaRouting(dagligReiseRoutingRequest)
+
+            assertThat(routingSjekk.skalBehandlesINyLøsning).isFalse()
+            assertThat(routingHarBlittLagret()).isFalse()
+        }
+
+        @Test
+        fun `skal route til gammel løsning hvis person har AAP men strengt fortrolig adresse`() {
+            mockMaksAntallSomKanRoutesPåDagligReise(maksAntall = 10)
+            mockAapVedtak(erAktivt = true)
+            mockPersonMedAdressebeskyttelse(AdressebeskyttelseGradering.STRENGT_FORTROLIG)
+
+            val routingSjekk = kall.søknadRouting.skjemaRouting(dagligReiseRoutingRequest)
```

Comment:
Ser jeg har bomma litt på navngivningen her, dette burde vel vært `kall.ekstern.skjemaRouting` for å matche opp mot url-en